docs/AusTrakka CLI/CLI-metadata-upload.md

```diff
@@ -43,26 +39,31 @@ values should also be listed in the extra tabs of the spreadsheet template.
 
 Metadata can be uploaded against a chosen proforma specification by running
 ```
-austrakka metadata add -p <proforma-abbreviation> <metadata-file>
+austrakka metadata add -p <proforma-abbreviation> --owner-org <org-abbreviation> --shared-projects <project-abbreviation> <metadata-file>
```

Comment:
Might be better to default instructions to the short forms (-oo, -sp) since the proforma flag is already set to the short form.